verification/src/transaction_verifier.rs

```diff
@@ -589,29 +659,36 @@ impl Since {
 pub struct SinceVerifier<'a, DL> {
     rtx: &'a ResolvedTransaction,
     data_loader: &'a DL,
+    proposal_windows: ProposalWindow,
     block_number: BlockNumber,
     epoch_number_with_fraction: EpochNumberWithFraction,
     parent_hash: Byte32,
+    tx_phase: TransactionVerificationPhase,
     median_block_count: usize,
     median_timestamps_cache: RefCell<LruCache<Byte32, u64>>,
 }
 
 impl<'a, DL: HeaderProvider> SinceVerifier<'a, DL> {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
There are 4 verifiers have `#[allow(clippy::too_many_arguments)]`. All of them have 4 same arguments, three of them have 7 same arguments.
Could do refactor later, in another PR :dog:.